### PR TITLE
add maximum username length & maximum/minimum space url length

### PR DESCRIPTION
**Pull request by rekollekt**
add maximum username length & maximum/minimum space url length for /protected/config/common.php

### CODE REVIEW

File: protected/humhub/modules/user/Module.php [hunk 1]
```diff
@@ -59,6 +59,11 @@ class Module extends \humhub\components\Module
      */
     public $adminCanChangeUserProfileImages = false;
 
+    /**
+     * @var int maximum username length
+     */
```
Comment (danielkesselberg) [inline protected/humhub/modules/user/Module.php]: Could you add `@since 1.3` here?

File: protected/humhub/modules/space/Module.php [hunk 2]
```diff
@@ -42,6 +42,16 @@ class Module extends \humhub\components\Module
      */
     public $disableFollow = false;
 
+    /**
+     * @var int maximum space url length
+     */
```
Comment (danielkesselberg) [inline protected/humhub/modules/space/Module.php]: Could you add `@since 1.3` here?

File: protected/humhub/modules/space/Module.php [hunk 3]
```diff
@@ -42,6 +42,16 @@ class Module extends \humhub\components\Module
...
+    public $maximumSpaceUrlLength = 45;
+
+    /**
+     * @var int minimum space url length
+     */
```
Comment (danielkesselberg) [inline protected/humhub/modules/space/Module.php]: Could you add `@since 1.3` here?

File: protected/humhub/modules/space/models/Space.php [hunk 4]
```diff
@@ -119,7 +119,7 @@ public function rules()
             [['visibility'], 'in', 'range' => [0, 1, 2]],
             [['visibility'], 'checkVisibility'],
             [['url'], 'unique', 'skipOnEmpty' => 'true'],
-            [['guid', 'name', 'url'], 'string', 'max' => 45, 'min' => 2],
+            [['guid', 'name', 'url'], 'string', 'max' => Yii::$app->getModule('space')->maximumSpaceUrlLength, 'min' => Yii::$app->getModule('space')->minimumSpaceUrlLength],
```
Comment (danielkesselberg) [inline protected/humhub/modules/space/models/Space.php]: MaximumSpaceUrlLength and minimumSpaceUrlLength are applied for guid, name and url. Could you move url to a seperate rule and leave guid and name?

**Comment by rekollekt** [comment]
please check all.

**Pull request merged**